[Getting Started Docs][A/B test] Create Snippets from install docs (#6397)
PostHog/posthog.com: contents/docs/integrate/_snippets/install-python.mdx

> <PythonInstall />

Reviewer: Should we combine these two Python Install components?

> ```bash

Reviewer: Starts abruptly compared to others, but not sure if because of the `<PythonInstall />` component above.

pip install posthog
```

In your app, import the `posthog` library and set your api key and host **before** making any calls.

```python
from posthog import Posthog

# The personal API key is necessary only if you want to use local evaluation of feature flags.
posthog = Posthog('<ph_project_api_key>', host='<ph_instance_address>', personal_api_key='<ph_personal_api_key>')
```

You can read more about the differences between the project and personal API keys in the dedicated [API authentication section](/docs/api/overview#authentication) of the Docs.

> > **Note:** As a general rule of thumb, we do not recommend having API keys in plaintext. Setting it as an environment variable would be best.

Reviewer: ```suggestion
> **Note:** As a rule of thumb, we do not recommend having API keys in plaintext. Setting it as an environment variable is best.
```

> You can find your keys in the 'Project Settings' page in PostHog.

Reviewer: ```suggestion
You can find your project API key and instance address in the [project settings](https://app.posthog.com/project/settings) page in PostHog.
```

> To debug, you can toggle debug mode on:

Reviewer: Why would someone want to use debug mode? See requests, errors in the output? We can replace "To debug" with this info.

```python
posthog.debug = True
```

> And to make sure no calls happen during your tests, you can disable them, like so:

Reviewer: ```suggestion
To make sure no calls happen during tests, you can disable PostHog, like so:
```

Reviewer: TIL
